src/site/layouts/home.drupal.liquid

```diff
@@ -0,0 +1,139 @@
+{% include "src/site/includes/header.html" %}
+<main>
+
+  <!-- the hub -->
+  <section class="homepage-hub">
+
+    <div class="homepage-hub-container">
+
+      <h1 class="heading-level-2 homepage-heading">Access and manage your VA benefits and health care</h1>
+
+      <!-- top row -->
+      <div class="hub-links-row">
+      {% for card in cards %}
+        <div class="hub-links-container">
+          <h2 class="heading-level-3 hub-links-title"><i class="icon-large-baseline icon-heading hub-icon-{{ card.label | downcase | replace: ' ', '-' }} hub-color-{{ card.label | downcase | replace: ' ', '-' }}"></i>{{ card.label }}</h2>
```

Comment:
Looks like the class list is different on this `<i>`. I'm not sure where `cards` comes from, so I'm having a hard time understanding the purpose for this change.

| home.html | home.drupal.liquid |
| --- | --- |
| `hub-icon-{{card.hub}}` | `hub-icon-{{ card.label \| downcase \| replace: ' ', '-' }}` |
| `hub-color-{{card.hub}}` | `hub-color-{{ card.label \| downcase \| replace: ' ', '-' }}` |

What's the difference between `card.hub` and `card.label`?

Reply:
Gotcha. Drupal / GraphQL with the `label` field is what I was looking for. :100: